src/worker/startup.ts

```diff
@@ -1,146 +1,17 @@
-	} catch (err) {
-		job.sentry.setExtra("job", {
-			id: job.id,
-			attemptsMade: job.attemptsMade,
-			timestamp: new Date(job.timestamp),
-			data: job.data
-		});
-
-		job.sentry.setTag("jiraHost", job.data.jiraHost);
-		job.sentry.setTag("queue", job.queue.name);
-		job.sentry.captureException(err);
-
-		throw err;
-	}
-};
-const logMiddleware = (jobHandler, jobName: string) => {
-	return async (job) => {
-		const jobLogger = logger.child({
-			name: jobName,
-			// Probot uses "id" key as "requestId" for tracing; let's use the same key for consistency and also
-			// to have uniform logs for webhooks that are processed both asynchronously and in request handlers
-			//
-			// Let's use random ID rather than "job.id" to be able to separate retries of the same job
-			id: uuidv4(),
-		});
-		try {
-			await jobHandler(job, jobLogger);
-		} catch (err) {
-			jobLogger.error({ err }, "Execution failed!");
-		}
-	};
-}
-const setDelayOnRateLimiting = (jobHandler) => async (job: Job, logger: LoggerWithTarget) => {
-	try {
-		await jobHandler(job, logger);
-	} catch (err) {
-		if (err instanceof RateLimitingError) {
-			// delaying until the rate limit is reset (plus a buffer of a couple seconds)
-			const delay = err.rateLimitReset * 1000 + 10000 - new Date().getTime();
-
-			if (delay <= 0) {
-				logger.warn({ job }, "Rate limiting detected but couldn't calculate delay, delaying exponentially");
-				job.opts.backoff = {
-					type: "exponential",
-					delay: 10 * 60 * 1000
-				};
-			} else {
-				logger.warn({ job }, `Rate limiting detected, delaying job by ${delay} ms`);
-				job.opts.backoff = {
-					type: "fixed",
-					delay: delay
-				};
-			}
-		}
-		throw err;
-	}
-};
-
-const sendQueueMetrics = async () => {
-	// Only send queue metrics in prod
-	if (!isNodeProd()) {
-		return;
-	}
-
-	for (const [queueName, queue] of Object.entries(queues)) {
-		logger.info("fetching queue metrics");
-
-		const jobCounts = await queue.getJobCounts();
-
-		logger.info({ queue: queueName, queueMetrics: jobCounts }, "publishing queue metrics");
-
-		const tags = { queue: queueName };
-		statsd.gauge(redisQueueMetrics.active, jobCounts.active, tags);
-		statsd.gauge(redisQueueMetrics.completed, jobCounts.completed, tags);
-		statsd.gauge(redisQueueMetrics.delayed, jobCounts.delayed, tags);
-		statsd.gauge(redisQueueMetrics.failed, jobCounts.failed, tags);
-		statsd.gauge(redisQueueMetrics.waiting, jobCounts.waiting, tags);
-	}
-};
 
-const commonMiddleware = (jobHandler, loggerName: string) => logMiddleware(sentryMiddleware(setDelayOnRateLimiting(jobHandler)), loggerName);
 let running = false;
-let timer: Timeout;
 
-// Start function for Node cluster worker
 export async function start() {
 	if (running) {
 		logger.debug("Worker instance already running, skipping.");
 		return;
 	}
 
 	logger.info("Micros Lifecycle: Starting queue processing");
-	// exposing queue metrics at a regular interval
-	timer = setInterval(sendQueueMetrics, 60000);
```

Comment:
do we still have charts with these metrics? Or are we emitting the same metrics with the new queues?